File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.cs
using System;
using System.Linq;

public class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        
        int min = lottos.Intersect(win_nums).Count();
    
        int max = lottos.Where(element => element == 0).Count();
        
        return new int[] { Math.Min(6, 7 - (max + min)) , Math.Min(6, 7 - (min)) };
    }
}

File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.js
function solution(lottos, win_nums) {
    lottos = lottos.sort((a,b)=> a - b)
    console.log(lottos)
    let max = 0, min = 0
    lottos.forEach((element)=>
    {
        element === 0 ? max++ : max;
        win_nums.forEach((element2)=>
        {
            element === element2 ? min++ : min;    
        })
    })
    
    return [7 - (max + min) === 7 ? 6 : 7 - (max + min),
            7 - (min) === 7 ? 6 : 7 - (min)];
}